gladier_tools/globus/transfer.py

```diff
@@ -0,0 +1,40 @@
+from gladier import GladierBaseTool
+
+
+class Transfer(GladierBaseTool):
```

Comment:
Just so I understand. You wrapped the globus transfer AP into a gladiertool so the clients can autogenerate the flows.
Correct?